hypervisor/common/hypercall.c

```diff
@@ -171,7 +171,7 @@ int32_t hcall_create_vm(struct acrn_vm *vm, uint64_t param)
 
 			/* post-launched VM is allowed to choose pCPUs from vm_config->cpu_affinity_bitmap only */
 			if ((cv.cpu_affinity & ~(vm_config->cpu_affinity_bitmap)) != 0UL) {
-				pr_err("%s: Post-launch VM can't share PCPU with Pre-launch VM!", __func__);
+				pr_err("Post-launched VM%u chooses invalid pCPUs(0x%llx).", vm_id, cv.cpu_affinity);
```

Comment:
This is fine.
Can you change the style to be  If (..) { main code} else {error processing} ?

It is vastly existed, but let gradually clean it when a new patch is merged.

Reply:
Sure, will fix it and use this style going forward.